Framework/TestHelpers/src/WorkspaceCreationHelper.cpp

```diff
@@ -504,8 +504,16 @@ createEventWorkspaceWithNonUniformInstrument(int numBanks, bool clearEvents) {
   return ws;
 }
 
+void addComponent(Instrument_sptr &instrument, const V3D &position,
+                  std::string name = "component") {
+  ObjComponent *component = new ObjComponent(name);
+  component->setPos(position);
+  instrument->add(component);
+}
+
 void addSource(Instrument_sptr &instrument, const V3D &position,
                std::string name = "source") {
+  addComponent(instrument, position, name);
```

Comment:
@reimundILL I think your problem is here - I guess this line was added accidentally!

Reply:
Thank you very much! On my machine that seems to solve the problem, let's see ...